resources/assets/css/_forms.css

```diff
@@ -19,7 +19,8 @@
 }
 
 .input-text {
-    @apply form-input px-4 py-3 block w-full transition-default;
+    @apply form-input block w-full transition-default;
+    padding: 0.75rem 1rem;
```

Comment:
can you add a line why this is needed so we don't change it at a later time

Reply:
Thanks for adding it 👍